src/main/java/org/folio/fqm/service/CrossTenantQueryService.java

```diff
@@ -68,7 +68,7 @@ public List<String> getTenantsToQuery(EntityType entityType, boolean forceCrossT
   @SuppressWarnings("unchecked")
   // JsonPath.parse is returning a plain List without a type parameter, and the TypeRef (vs Class) parameter to JsonPath.read is not supported by the JSON parser
   private List<Map<String, String>> getUserTenants() {
-    String userTenantsResponse = ecsClient.get("user-tenants", Map.of("limit", "1000"));
+    String userTenantsResponse = ecsClient.get("user-tenants", Map.of("userId", executionContext.getUserId().toString(), "limit", "1000"));
```

Comment:
I probably should have noticed this was missing when I reviewed the previous one... 😬

Reply:
Turns out it shouldn't be there anyway 😛 In member tenants, there's only a dummy user with a random userId, so adding a `userId` query param makes it filter the dummy user out. However, we need that dummy user to get the central tenant ID and consortium ID